pmd-java/src/main/resources/category/java/errorprone.xml

```diff
@@ -697,7 +697,7 @@ private int getMoreFoo(){
     <rule name="BrokenNullCheck"
           language="java"
           since="3.8"
-          message="Method call on object which may be null"
+          message="This expression will throw a NullPointerException"
```

Comment:
I changed this because we now report on other kinds of expressions than just method calls (all QualifiableExpressions + operands that are unboxed)